security.php

```diff
@@ -54,56 +57,196 @@ function wpcom_vip_is_restricted_username( $username ) {
 		|| WPCOM_VIP_MACHINE_USER_EMAIL === $username;
 }
 
+/**
+ * PRIVATE: This function is not intended to be used outside of this file and is subject to change without notice.
+ *
+ * Returns the defualts for limits.
+ */
+function _vip_get_auth_count_limit_defaults() {
```

Comment:
@rinatkhaziev this solution combines both the default value, which we could later sourced in a different manner as well as a set of filters for code adjustment.

Even if we decide not to implement different sourcing, it's still handy to have these values here in one place instead of spread out over the code below.

What do you think?